File: wallet_syncronizer/pkg/database/wallet/wallet.go
package wallet

import "time"

func (Wallet) TableName() string {
	return "Wallet"
}

type Wallet struct {
	WalletId  string    `json:"wallet_id" gorm:"column:WalletId;primaryKey"`
	CreatedAt time.Time `json:"created_at" gorm:"column:CreatedAt;autoCreateTime"`
}

File: wallet_syncronizer_node/database/wallet.js
const { DataTypes } = require('sequelize');

module.exports = (sequelize) => {
    return sequelize.define('Wallet', {
        WalletId: {
            type: DataTypes.STRING,
            primaryKey: true,
        },
        CreatedAt: {
            type: DataTypes.DATE,
            defaultValue: DataTypes.NOW,
        },
    }, {
        tableName: 'Wallet',
        timestamps: false, // Puoi impostare questo su true se desideri timestamps
    });
};
